scrapyd/default_scrapyd.conf

```diff
@@ -19,6 +19,7 @@ webroot     = scrapyd.website.Root
 [services]
 schedule.json     = scrapyd.webservice.Schedule
 cancel.json       = scrapyd.webservice.Cancel
+status.json       = scrapyd.webservice.JobStatus
```

Comment:
better name the endpoint jobstatus to be more explicit
and consistent with other ws names

Reply:
We don't call the others `jobschedule` or `jobcancel` so I think I'll return this to `status`.